release.yaml

```diff
@@ -28,3 +28,7 @@ rancher-cis-benchmark:
   - 6.3.0-rc.1
 rancher-cis-benchmark-crd:
   - 6.3.0-rc.1
+rancher-sachet:
```

Comment:
If we are removing versions why are we adding them to release.yaml?

Reply:
Weird - maybe an issue with the remove command? I just did:

```
CHART=rancher-sachet VERSION=* make remove
```

Sequentially for each affected version.

Reply:
Not sure the cause - but I've corrected it now.